R/updateGateFamily.R

```diff
@@ -0,0 +1,24 @@
+#' Update gate family
+#'
+#' Updates a gate family.
+#'
+#' @param experimentId ID of experiment.
+#' @param gid Gate family ID
+#' @param properties Properties to set on the experiment.
```

Comment:
s/experiment/gate/

Reply:
What are you referring to with "s/experiment/gate/" here?

Reply:
like the vim replace command, sorry -- replace "experiment" with "gate" in the doc line.